chromadb/api/configuration.py

```diff
@@ -319,34 +346,32 @@ def from_legacy_params(cls, params: Dict[str, Any]) -> Self:
 # Internally, we pass around HNSWConfigurationInternal objects, which perform
 # validation, serialization and deserialization. Users don't need to know
 # about that and instead get a clean constructor with default arguments.
-class HNSWConfigurationInterface(HNSWConfigurationInternal):
+def HNSWConfigurationInterface(
```

Comment:
this is a bit odd to me being a func

Reply:
This follows the pattern we have for users creating `Client`; this is an interface in the sense that it doens't actually represent a new object, it just gives you an easy way of creating one that can be user-facing.